src/open_inwoner/templates/pages/search.html

```diff
@@ -20,10 +20,26 @@ <h1 class="h1">Zoeken</h1>
         {#  facets and results section #}
         {% if results %}
             <div class="grid">
-                <aside class="grid__sidebar" aria-label="{% trans "Zoekfilters" %}">
-                    {% filter field=form.categories %}
-                    {% filter field=form.tags %}
-                    {% filter field=form.organizations %}
+                <aside class="grid__sidebar grid__filters" aria-label="{% trans "Zoekfilters" %}">
+                    <h2 class="h2">{% trans "Zoekfilters" %}</h2>
```

Comment:
This title stays even when I disable all the search options, not sure if that is intented?